### ISSUE TITLE
Make a customisable AdView using the same technology

### ISSUE THREAD

**Issue by Pradyuman7** [Pradyuman7/LookAtMe#3]
Make a clear, customisable android AdView using the same concepts

An AdView would definitely be a great way to show ads to the user and know if the user really saw the ad or just flipped it for nothing.

**Comment by kevin-zhan** [comment]
I don't think I would let an app spy me when I'm using it, especially the app want to know whether I focus on its ads...

**Comment by kevin-zhan** [comment]
Of course, as a developer, I do think it's a great lib. But as a user, I'll never use any app with this function.

**Comment by Pradyuman7** [comment]
@kevin-zhan I understand the concern but every development has its use cases, like for this I think that apps which provide money for watching ads can be more sure of an engaging user if they use such technology. I also understand that this might be very troublesome and annoying for users but a good balance between this, might be better... 😃

**Comment by yudikarma** [comment]
can i use this feature for other view like implement for read manga and other reading book ?

**Comment by Pradyuman7** [comment]
@yudikarma Currently LookAtMe just supports VideoView, but I would love to work on reading book like view or something.
